README.md

```diff
@@ -6,8 +6,8 @@ This repository contains tools to deploy testing or development two-node Openshi
 These tools expect a valid RHEL hypervisor to host all necessary VMs to create and set up the cluster.
 If you don't already have one available, the [dev-env-aws-hypervisor](deploy/dev-env-aws-hypervisor/) folder provides instructions and tools to automatically create this base host in AWS.
 
-Once the hypervisor is setup, you can use the [TNA](deploy/tna-ipi-baremetalds-virt/) and TNF folders to create those specific clusters within that newly provisioned host, or on your own one. 
+Once the hypervisor is setup, you can use the [TNA](deploy/ipi-baremetalds-virt/) and TNF folders to create those specific clusters within that newly provisioned host, or on your own one. 
```

Comment:
This sentence seems to imply there there should be 2 folders. Can we tweak it to (1) point us to the base automation dir, and (2) link us to the specific instructions for TNA and TNF respectively?